addons/vanguard/CfgGroups_Opfor.hpp

```diff
@@ -0,0 +1,389 @@
+class EAST {
+    class TACU_Vanguard_O {
+        name = "Vanguard PMC";
+        class TACU_Vanguard_G_O_Contractors_Green {
+            name = "Contractors (Green)";
+
+            class TACU_Vanguard_G_Contractors_Green_Patrol {
+                name = "Patrol";
+                side = 0;
+                faction = "TACU_Vanguard_O";
+                icon = "\a3\ui_f\data\map\markers\nato\n_inf.paa";
```

Comment:
```suggestion
                icon = "\a3\ui_f\data\map\markers\nato\o_inf.paa";
```
Opfor is `o_inf.paa` (and blufor `b_inf.paa` in case you were curious).